images/regfile.proto

```diff
@@ -4,13 +4,17 @@ import "opts.proto";
 import "fown.proto";
 
 message reg_file_entry {
-	required uint32		id	= 1;
-	required uint32		flags	= 2 [(criu).flags = "rfile.flags"];
-	required uint64		pos	= 3;
-	required fown_entry	fown	= 5;
-	required string		name	= 6;
-	optional sint32		mnt_id	= 7 [default = -1];
-	optional uint64		size	= 8;
-	optional bool		ext	= 9;
-	optional uint32         mode    = 10;
-}
+	required uint32		id			= 1;
+	required uint32		flags			= 2 [(criu).flags = "rfile.flags"];
+	required uint64		pos			= 3;
+	required fown_entry	fown			= 5;
+	required string		name			= 6;
+	optional sint32		mnt_id			= 7 [default = -1];
+	optional uint64		size			= 8;
+	optional bool		ext			= 9;
+	optional uint32         mode    		= 10;
+	repeated uint32 	build_id 		= 11;
```

Comment:
I don't think that we need to add all of them in one commit. I think it is better to add a field together with the implementation of its check. And you can create a separate PR for each check. It is easier to review small PR-s.

Reply:
In that case, I'll use this for the CLI options and build-ID?

Reply:
I don't understand what you mean. Could you elaborate?

Reply:
> I don't think that we need to add all of them in one commit. I think it is better to add a field together with the implementation of its check. And you can create a separate PR for each check. It is easier to review small PR-s.

I'll implement the build-ID check in this PR along with the command line arguments

Reply:
yes, this is right.